python/rikai/spark/functions/vision.py

```diff
@@ -16,6 +16,8 @@
 """
 
 # Third Party
+import ffmpeg
```

Comment:
Hi, is `ffmpeg` a required dependency of rikai or an optional one？
Should we do lazy import of `ffmpeg` if this is not a required dependency, so that it does not leak to non-video cases.

Reply:
That sounds good.